nxcompshad/src/X11.cpp

```diff
@@ -363,7 +363,7 @@ void Poller::shmInit(void)
 
     shminfo_ -> shmaddr = (char *)shmat(shminfo_ -> shmid, 0, 0);
 
-    if (shminfo_ -> shmaddr < 0)
+    if (shminfo_ -> shmaddr < (void *)0)
```

Comment:
Why not just checking for `< NULL`?

Reply:
I found [this](http://www.stroustrup.com/bs_faq2.html#null)

Reply:
Yeah... In C, `NULL` is typically defined as `(void*)0`, but this doesn't seem to be legit in C++ for some reason.

Thus using `(void*)0` here is absolutely fine, you're right.